connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/Constants.java

```diff
@@ -415,6 +416,7 @@ public void marshallAsElement(AttributeDefinition attribute, ModelNode resourceM
                     AUTHENTICATION_CONTEXT_AND_APPLICATION_NAME)
             .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.SECURITY_DOMAIN_REF)
             .addAccessConstraint(ResourceAdaptersExtension.RA_SECURITY_DEF)
+            .setDeprecated(ELYTRON_BY_DEFAULT_VERSION)
```

Comment:
This is already there a couple lines down.

Reply:
fixed